recipe/meta.yaml

```diff
@@ -1,29 +1,26 @@
-{% set name = "pywin32" %}
-{% set version = "223" %}
+{% set version = "226" %}
```

Comment:
I did not see a 226 in the repo but it looks like it was tagged.

Reply:
Strange, I don't see the tag: https://github.com/mhammond/pywin32/tags